scripts/astrokat-cals.py

```diff
@@ -600,9 +736,17 @@ def main(args):
                                 )
```

Comment:
'target' on line 775 redefines definition on line 727

Reply:
Yes, I know, flake8 reminds me of this

This is intentional, since the input targets may have spaces added by users, this redefinition simply removes the spaces